main/config/settings.c

```diff
@@ -380,6 +391,7 @@ static const char *view_crsf_input_tx_settings[] = {
     SETTING_KEY_ABOUT,
     SETTING_KEY_ABOUT_VERSION,
     SETTING_KEY_ABOUT_BUILD_DATE,
+    SETTING_KEY_ABOUT_MAC,
```

Comment:
I think it would be better to rename `SETTING_KEY_ABOUT_MAC ` to `SETTING_KEY_ABOUT_ADDR`, the latter is a bit more generic and not all platforms will have a proper MAC available.